source/includes/AdministrationLinks/administration_links.md

```diff
@@ -0,0 +1,170 @@
+# Administration Links
+
+## Administration Links Overview
+
+Every installed module can register its administration pages. They are accessible through Legacy UI at [openmrs/admin/index.htm](openmrs/admin/index.htm)
+
+The following endpoints allow users to return list of installed modules with their administration links.
+
+## Available Operations for Administration Links type
+
+1. [List Administration Links](#list-administration-links)
+
+## List Administration Links
+
+> List Administration Links
+
+```shell
+GET /administrationlinks?v=default
+```
+
+```java
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+Request request = new Request.Builder()
+  .url("/openmrs/ws/rest/v1/administrationlinks?v=default")
+  .method("GET", null)
+  .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+  .addHeader("Cookie", "JSESSIONID=DF385B2E6E39E0BB49BB7E079BF31C44")
+  .build();
+Response response = client.newCall(request).execute();
+```
+
+```javascript
+var myHeaders = new Headers();
+myHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+myHeaders.append("Cookie", "JSESSIONID=DF385B2E6E39E0BB49BB7E079BF31C44");
+
+var requestOptions = {
+  method: 'GET',
+  headers: myHeaders,
+  redirect: 'follow'
+};
+
+fetch("/openmrs/ws/rest/v1/administrationlinks?v=default", requestOptions)
+  .then(response => response.text())
+  .then(result => console.log(result))
+  .catch(error => console.log('error', error));
+```
+
+> Success Response
+
+```response
+{
+    "results": [
+        {
+            "uuid": "webservices.rest",
+            "display": "webservices.rest.title",
+            "title": "webservices.rest.title",
+            "administrationLinks": {
+                "module/webservices/rest/settings.form": "webservices.rest.manage.settings",
```

Comment:
I guess we are gonna change these to reflect the actual values pointed to by the properties.